packages/vector_graphics_codec/lib/vector_graphics_codec.dart

```diff
@@ -278,6 +286,12 @@ class VectorGraphicsCodec {
     } else {
       buffer._putInt32(0);
     }
+    if (transform != null) {
+      buffer._putInt32(transform.length);
```

Comment:
here and above: the transform is always going to either be 0 or 16 entries, we should be able to use a Uint8 here.